pyomo/contrib/fbbt/expression_bounds_walker.py

```diff
@@ -280,4 +280,7 @@ def beforeChild(self, node, child, child_idx):
         return _before_child_handlers[child.__class__](self, child)
 
     def exitNode(self, node, data):
-        return _operator_dispatcher[node.__class__](self, node, *data)
+        if node.__class__ in _operator_dispatcher:
+            return _operator_dispatcher[node.__class__](self, node, *data)
+        else:
+            return -inf, inf
```

Comment:
Should this at least toss a warning that we are giving up on bounds tightening because of an unknown / unhandled operator?

Reply:
(should we also update the other FBBT walkers to also emit warnings for unknown expression types?)

Reply:
If I recall correctly, we used to have warnings in there, but they were annoying, so we removed them.

Reply:
I think that it is OK to return (-inf, inf) bounds with no warning for certain known expression nodes where that is all we can do (e.g., ExternalFunctionExpression).  But when we hit expression nodes that we just weren't expecting / never thought about, I think we should issue a warning: that way a) the user's code doesn't up and fail, and b) they file an issue so that we *do* think about it a little and intentionally decide how to handle it.